python_modules/libraries/dagster-openai/dagster_openai/resources.py

```diff
@@ -180,6 +180,9 @@ def openai_asset(context: AssetExecutionContext, openai: OpenAIResource):
     """
 
     api_key: str = Field(description=("OpenAI API key. See https://platform.openai.com/api-keys"))
+    organization: Optional[str] = None
+    project: Optional[str] = None
+    base_url: Optional[str] = None
```

Comment:
Could we use `pydantic.Field` here, like for `api_key`? You could set a default value like [this](https://docs.pydantic.dev/latest/concepts/fields/#default-values).

Reply:
Absolutely -- updated in https://github.com/dagster-io/dagster/pull/23260/commits/bea7a4a96ecd3077cf35a9b899cd82970d98350c ✅